app/src/main/java/io/pslab/fragment/BaroMeterDataFragment.java

```diff
@@ -604,7 +604,7 @@ private void resetInstrumentData() {
     }
 
     private void initiateBaroSensor(int type) {
-        BaroMeterDataFragment.BARO_SENSOR s = BaroMeterDataFragment.BARO_SENSOR.values()[type];
```

Comment:
Revert this.

Reply:
@bessman @CloudyPadmal 
Sir please help me how t fix this problem

Reply:
Change the line back to what it was before.

Reply:
> Change the line back to what it was before.

Sir i change these two line. But it is not work.How to fix it please help me.